enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotChatEmbedding.tsx

```diff
@@ -102,27 +105,15 @@ export const MetabotChatEmbedding = ({
     dispatch(resetConversationId());
   }, [dispatch]);
 
+  useEffect(() => {
+    const normalizedMessages = metabot.userMessages.filter(
+      (message) => message !== METABOT_RESULTS_MESSAGE,
```

Comment:
there's no way to differentiate between different kinds of reactions yet. [Ref](https://metaboat.slack.com/archives/C07SJT1P0ET/p1745579017937989?thread_ts=1745577165.764459&cid=C07SJT1P0ET)

Since we don't want to show this, we need to hard code it.

[The design](https://www.figma.com/design/l9y9OEENQUYiKSbFo7WQA0/Embedding-Metabot-w--SDK-to-ask-questions-using-natural-language?node-id=17229-10220&t=1IYfhQTGaiwd6XTz-0)

Reply:
yes, so there these discussions scatter in a few different places, so I couldn't really pinpoint them sorry about that.

we don't wanna display the success message ("Here are the results") and just show the result outright. And for other messages we just show them as an agent response/notification/error instead of a bubble.